### PR TITLE
Not showing private account name in account menu anymore.

### LINKED ISSUE #1979: Use placeholder name for anonymous accounts
Instead of the generated private id, display something like "Anonymous Account" as the user name. Otherwise, people might get confused as to why they appear logged in despite never creating an account.

### PR DESCRIPTION
**Pull request by peacememories**
Fixes #1979

### CODE REVIEW

File: webofneeds/won-owner-webapp/src/main/webapp/app/components/account-menu.js [hunk 1]
```diff
@@ -77,7 +77,11 @@ function genLogoutConf() {
     }
 
     getEmail() {
-      return ellipsizeString(this.email, this.maxEmailLength);
+      if (this.email.endsWith("matchat.org")) {
```
Comment (quasarchimaere) [inline webofneeds/won-owner-webapp/src/main/webapp/app/components/account-menu.js]: i think we could use privateIdUser instead (is stored in the state)

Comment (peacememories) [reply webofneeds/won-owner-webapp/src/main/webapp/app/components/account-menu.js]: I looked into the state and tehre doesn't seem to be anything like that?

Comment (quasarchimaere) [reply webofneeds/won-owner-webapp/src/main/webapp/app/components/account-menu.js]: nvm i just saw that i use the routeParam instead...e.g. topnav.js " isPrivateIdUser: !!privateId," should work too though not sure if i like it better than the email ending though...i guess noone will have a user with a matchat.org email anyway... maybe we just need to mark it with a todo for if we ever need to create privateIdUsers from a different domain though

Comment (peacememories) [reply webofneeds/won-owner-webapp/src/main/webapp/app/components/account-menu.js]: I extracted the call into a selector on the state. This should make it more reusable.

**Comment by peacememories** [comment]
In the meantime, can you resolve your change request? :)

**Comment by quasarchimaere** [comment]
sorry... :-(

**Pull request merged**